packages/node-image-compute/base.packages

```diff
@@ -1,9 +1,110 @@
-# Packages common to all mediums.
```

Comment:
Investigate `spire-agent`, we install this in `node-images-base` from the `_ncn` repositories. Look into the `_cn` counterpart, see what is different (if anything).

`spire-agent` is required by `cray-auth-utils`, which may be required by other packages here - but these would pull `spire-agent` from the `_cn` repo.

Reply:
We do have a ticket to transfer `cray-auth-utils`.